crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -0,0 +1,177 @@
+//! A [`bevy_picking`] backend for sprites.
+
+use std::cmp::Ordering;
+
+use crate::{Sprite, TextureAtlas, TextureAtlasLayout};
+use bevy_app::prelude::*;
+use bevy_asset::prelude::*;
+use bevy_ecs::prelude::*;
+use bevy_math::prelude::*;
+use bevy_picking::backend::prelude::*;
+use bevy_render::prelude::*;
+use bevy_transform::prelude::*;
+use bevy_window::PrimaryWindow;
+
+#[derive(Clone)]
+pub struct SpriteBackend;
```

Comment:
Could be helpful to add picking to the name, if I saw `SpriteBackend` I wouldn't think it was for picking. Not sure if there are any established conventions for this yet.